Compiling fails

Trying to compile it after following all instructions, fails when running `make`. I get the following error:
```
gcc -std=c99 -Wall -Wextra -Wno-maybe-uninitialized -pedantic -Wmissing-prototypes -Wstrict-prototypes -O3 -march=native -fomit-frame-pointer  -DED25519_amd64_51_30k  -DSTATISTICS -c -o main.c.o.tmp main.c && mv main.c.o.tmp main.c.o
main.c: In function ‘main’:
main.c:483:13: error: ‘checkpointfile’ undeclared (first use in this function)
  483 |         if (checkpointfile && !deterministic) {
      |             ^~~~~~~~~~~~~~
main.c:483:13: note: each undeclared identifier is reported only once for each function it appears in
main.c:483:32: error: ‘deterministic’ undeclared (first use in this function)
  483 |         if (checkpointfile && !deterministic) {
      |                                ^~~~~~~~~~~~~
make: *** [GNUmakefile:132: main.c.o] Error 1
```

Am I missing some kind of libraries?
gcc: `gcc version 11.2.0 (Debian 11.2.0-18)`

--- a/main.c
+++ b/main.c
@@ -480,10 +480,12 @@ int main(int argc,char **argv)
 		exit(1);
 	}
 
+#ifdef PASSPHRASE
 	if (checkpointfile && !deterministic) {
 		fprintf(stderr,"--checkpoint requires passphrase\n");
 		exit(1);
 	}
+#endif
 
 	if (outfile) {
 		fout = fopen(outfile,!outfileoverwrite ? "a" : "w");
@@ -661,7 +663,7 @@ int main(int argc,char **argv)
 			perror("pthread_attr_destroy");
 	}
 
-#if PASSPHRASE
+#ifdef PASSPHRASE
 	pthread_t checkpoint_thread;
 
 	if (checkpointfile) {

